### PR TITLE
10.0 mig base user reset access mgs

### PR DESCRIPTION
**Pull request by GillesTephaneMeyomesse**

### CODE REVIEW

**Comment by sbidoul** [comment]
@GillesTephaneMeyomesse @adrienpeiffer Do we still need this module? In my view, base_user_role covers this functionality and more.

File: base_user_reset_access/README.rst [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+======================
+
+This module adds a button to reset the user's access rights to the minimum configuration.
+
+Installation
+============
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/149/8.0
```
Comment (JayVora-SerpentCS) [inline base_user_reset_access/README.rst:16]: Runbot 10.0

File: base_user_reset_access/README.rst [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/server-tools/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed feedback
+`here <https://github.com/OCA/server-tools/issues/new?body=module:%20base_user_reset_access%0Aversion:%208.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
```
Comment (JayVora-SerpentCS) [inline base_user_reset_access/README.rst:25]: Update the bugtracker section as per https://raw.githubusercontent.com/OCA/maintainer-tools/master/template/module/README.rst

File: base_user_reset_access/README.rst [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+.. image:: http://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: http://odoo-community.org
+
+This module is maintained by the OCA.
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose mission is to support the collaborative development of Odoo features and promote its widespread use.
+
+To contribute to this module, please visit http://odoo-community.org.
```
Comment (JayVora-SerpentCS) [inline base_user_reset_access/README.rst:46]: https everywhere

File: base_user_reset_access/__init__.py [hunk 4]
```diff
@@ -0,0 +1 @@
+from . import models
```
Comment (JayVora-SerpentCS) [inline base_user_reset_access/__init__.py:1]: License and utf-8 text plz

File: base_user_reset_access/models/res_users.py [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+from odoo import models, api, exceptions, _
+from odoo.tools import SUPERUSER_ID
+
+
+class ResUsers(models.Model):
+    _inherit = 'res.users'
+
+    @api.multi
+    def reset_access_right(self):
```
Comment (JayVora-SerpentCS) [inline base_user_reset_access/models/res_users.py:13]: I have a possible doubt. We should also check whether the current method is being used by SUPERUSER_ID, possible threat is any user can access the method and change anyone's config?
Haven't checked but its worth trying.

**Comment by JayVora-SerpentCS** [comment]
@sbidoul is right, we should make a mix of both modules and base_user_role to be only used. There should still be a button for those firms who avoid using roles.
https://github.com/OCA/server-tools/tree/10.0/base_user_role

**Comment by JayVora-SerpentCS** [comment]
need to even merge transifex commits

**Comment by sbidoul** [comment]
I'm not sure merging this with base_user_role makes sense.

For our use cases, base_user_role provides the necessary reset mechanism.
We will therefore not continue this PR, so closing. Anyone feel free to take over.

**Pull request closed without merging**